sdk/src/system_impl.cpp

```diff
@@ -92,7 +92,7 @@ SystemImpl::getCameraList(std::vector<std::shared_ptr<Camera>> &cameraList,
 
     cameraList.clear();
 
-    if (uri.compare(0, 3, "ip:") == 0) {
+    if (uri.compare(0, 3, "IP:") == 0) {
```

Comment:
it would make more sense to downcast the entire string so we can support any variation (ip, IP, iP, Ip)